uint16      uid       
string      name
NodeStatus  prev_status 
NodeStatus  status
time        timestamp
